java-hello-world-maven/README-Cloud-Shell.md

```diff
@@ -74,17 +89,21 @@ This step is optional - [Check software version and environment variables](./REA
     Hello World!
     ```
 
-4. Let's use GraalVM Native Image to produce a native executable.
+## Step 5: Build and run as a native executable
+
+Let's use GraalVM Native Image to produce a native executable.
+
+**Option 1: Quick Build enabled**
 
-    4.1) **Option 1: Quick Build enabled**
+You will notice the `Quick Build` mode reduces the time required to generate a native executable, making it covenient to use Native Image builds in a typical development cycle (edit, compile, test, and debug). The `Quick Build` mode is recommended for development purpose only.
 
-    To enable `Quick Build`, open [pom.xml](./pom.xml) in a text editor like Nano and uncomment the line shown:
+1. To enable `Quick Build`, open [pom.xml](./pom.xml) in a text editor like Nano and uncomment the line shown:
 
     ```
     <buildArg>-Ob</buildArg>
     ```
 
-    Run the Native Image build to generate a native executable:
+2. Run the Native Image maven build to generate a native executable:
```

Comment:
```suggestion
2. Run the Native Image maven build to generate a native executable:
```
This sentence is a bit clumsy. You could try "Use the Native Image maven plugin to  create a native executable"

Reply:
Done.